### DISCUSSION TITLE
installing chia v1.2.5 on freebsd (truenas)

### DISCUSSION THREAD

**Discussion by gosmarty** [Chia-Network/chia-blockchain#8361] [Q&A]
Hi there,
since version 1.2.4 I'm unable to use chia on a freebsd system. I think the problem may be with the keychain.py file which is completely new. So, the instructions in the wiki to change some parts did not work anymore. I still get chia compiled but after that it's not working.
Does anybody have a suggestion how to solve my problem?

**Comment by BrandtH22** [comment]
@gosmarty , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

Note - freebsd specific instructions can be found here - https://docs.chia.net/installation#other-environments

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
